kubernetes-ingress-defaultbackend.yaml

```diff
@@ -38,7 +38,19 @@ update:
     strip-prefix: v
 
 test:
+  environment:
+    contents:
+      packages:
+        - wget
+        - wait-for-it
   pipeline:
     - name: help message
       runs: |
         defaultbackend --help
+    - name: Basic functional test for endpoints
+      runs: |
+        defaultbackend &
```

Comment:
thank you, the tests look good.
Can we use `test/daemon-check-output` pipeline in here? If possible then we should default to using that and then check some logs and then in the post: block make use curl to make some request to the server.

Reply:
@kranurag7 the reason i didn't use `test/daemon-check-output` is because after running `defaultbackend -port=9091` we don't see any output logs in the terminal and due to which i didn't made use of it and went straight ahead with `defaultbackend &`
```
yash-pimple:~/os$ defaultbackend -port=9091


yash-pimple:~/os$^C
```